lib/network/NetworkUtil.js

```diff
@@ -1,20 +1,21 @@
 let util = require("../util");
 
 /**
- * @class NetworkUtil
+ * Utility Class
  */
 class NetworkUtil {
   /**
-   * @constructor NetworkUtil
+   * @inheritDoc
    */
   constructor() {}
 
   /**
    * Find the center position of the network considering the bounding boxes
    *
-   * @param {Array<vis.Node>} allNodes
-   * @param {Array<vis.Node>} [specificNodes=[]]
+   * @param {Array.<Node>} allNodes
+   * @param {Array.<Node>} [specificNodes=[]]
    * @returns {{minX: number, maxX: number, minY: number, maxY: number}}
+   * @static
```

Comment:
This is entirely correct, but I'm wondering if the various jsdoc tools can figure `static` out themselves from the function signatures....